### DISCUSSION TITLE
Filter showing matching fonts in viewing area

### DISCUSSION THREAD

**Discussion by gitlocaluser** [character-map-uwp/Character-Map-UWP#303] [Ideas]
Great app - super excited I can see icon fonts :D 

Addition to the app ( maybe I'm missing it ) would be an option that when using the "Search by name or unicode hex" , the keywords are the fonts presented in the large view area.  Currently you can only see those items in the dropdown making it hard to see what icons are available for say "add"

Thanks!

**Comment by JohnnyWestlake** [comment]
There will be very basic support for this in the next update in the main font window by typing in `char: ` followed by your query in the font list search box. 

i.e, to find all fonts containing `←` & `↗` you would type in `char: ←↗`.

![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/2319473/c1b6740e-9ade-4823-8b7e-efe01dda4487)

I will try to work out a good UI for exposing this functionality in the update after, including support for Compare fonts view.


(Similar to #290 )

**Comment by JohnnyWestlake** [comment]
Implemented in the latest build, details here: https://github.com/character-map-uwp/Character-Map-UWP/pull/306

Does this suit your needs for now?

**Reply by gitlocaluser** [reply]
![2023-12-02_20-19-28](https://github.com/character-map-uwp/Character-Map-UWP/assets/96404834/effc8834-f38a-4d19-8304-e54f1f1f4497)
maybe, cant tell from your screenshots but my ask is like my picture, I want the icons I searcg fo  iin the searcg box to show in the Private Use Area not the combobox
